### ISSUE TITLE
Lighthouse_sample data, inconsistency between date_test_string and date_test

### ISSUE THREAD

**Issue by neilsycamore** [sanger/lighthouse#245]
PAM (Ian W) have noticed data inconsistency in mlwh lighthouse_sample table where there is a date_test_string entry but the date_tested is null. Looking at the examples (877)  the date_test_string in the plate maps has a whitespace!

In addition (but date related) date_test_strings with format '19/07/2020 21:41' result in NULL date_test entries

RT: https://rt.sanger.ac.uk/Ticket/Display.html?id=710743

PAM's SQL
select * from lighthouse_sample where date_tested_string IS NOT NULL AND date_tested IS NULL order by created_at desc;
=> 877 records (+ve, -ve, void results)

Slack conversation with @pjvv 

- The dates format is one issue which we need a feature and data fix for

- The AP ones from May last year look to be a bug in the code which has been fixed so instead of setting the value to NULL is was set to an empty string “”. This could be fixed with a patch.

- The newer ones sometimes have double space between date and time and then space at the end so we need to clean these up before parsing them.
**So, in summary:**
1. We need to add a new datetime format for potential dates from the labs
2. Data patch to set all date_tested_string to NULL where they are an empty string
3. Clean the date tested string from the labs with regex before attempting parse
